roles/variables/defaults/main.yml

```diff
@@ -404,6 +407,17 @@ zookeeper_custom_properties: {}
 ### Boolean used for disabling of systemd service restarts when rootless install is executed
 zookeeper_skip_restarts: "{{ skip_restarts }}"
 
+### kafka controller variables
+
+### Boolean to configure controller with TLS Encryption. Also manages Java Keystore creation
+kafka_controller_ssl_enabled: "{{ssl_enabled}}"
+
+### Deprecated- Boolean to enable mTLS Authentication on controller (Server to Server and Client to Server). Configures kafka to authenticate with mTLS.
```

Comment:
Lets avoid any deprecated variables here.

Reply:
It was a mistake in copying comment. Property is not deprecated